Makefile

```diff
@@ -131,16 +133,23 @@ clean-ui:
 .PHONY: clean
 clean: clean-ui clean-backend ## Clean all build artifacts
 
-bin/golangci-lint: bin/golangci-lint-${GOLANGCI_VERSION}
-	@ln -sf golangci-lint-${GOLANGCI_VERSION} bin/golangci-lint
-bin/golangci-lint-${GOLANGCI_VERSION}:
-	@mkdir -p bin
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | bash -s -- -b ./bin/ v${GOLANGCI_VERSION}
+$(BIN_DIR):
+	@mkdir -p $(BIN_DIR)
+
+GOLANGCI_BIN := $(BIN_DIR)/golangci-lint
+GOLANGCI_CONFIG := $(ROOT_DIR)/.golangci.yml
+GOLANGCI_VERSION := 1.52.2
+
+bin/golangci-lint: bin/golangci-lint-$(GOLANGCI_VERSION)
+	@ln -sf golangci-lint-$(GOLANGCI_VERSION) bin/golangci-lint
+
+bin/golangci-lint-$(GOLANGCI_VERSION): $(BIN_DIR)
```

Comment:
For directories please use pipe target dependencies so that it doesn't cause this to re-run if the directory timestamp changes, for example:
```
bin/golangci-lint-$(GOLANGCI_VERSION): | $(BIN_DIR)
```
See Order Only dependencies here: https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html

Reply:
Fixed.